packages/charts/src/charts/pie/pie-slice.component.tsx

```diff
@@ -32,7 +32,7 @@ type Props = {
   autocolor: boolean;
   labelPosition: [number, number];
   activePosition: [number, number];
-  label: string;
+  label: string | number | Date | boolean;
```

Comment:
Do we know about any use cases when `Date` is provided as a label ?

Reply:
Date can come from `formatValue` function